.storybook/StorybookComponents/ColorList/index.styles.tsx

```diff
@@ -0,0 +1,14 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  colorGrid: ({ spacings }: Theme) =>
+    css({
+      display: 'grid',
+      gap: `${spacings.DOUBLE}rem`,
+      gridTemplateColumns: 'repeat(3, 1fr)',
```

Comment:
I'm wondering if we should include media queries to make the colour cards more responsive? 

If I view these storybook files in mobile view then they require horizontal scrolling, or reduce down to 3 across. I can see we're following a similar pattern to webcore.
[Scrollling (Our permalink)](https://ux_colour_documentation--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=docs-ux-colors-cross-platform--page&viewMode=story)
[Scrolling (webcore)](https://static.files.bbci.co.uk/core/storybook/index.html?path=/docs/foundations-colours-colour-tokens-cross-platform-colours--docs)

[Shrinks (our permalink)](https://ux_colour_documentation--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=docs-ux-colors-foundation--page&viewMode=story)
[Shrinks (webcore)](https://static.files.bbci.co.uk/core/storybook/index.html?path=/docs/foundations-colours-colour-tokens-foundation-colours--docs)

It's a bit of a nitpick, since it's unlikely anyone will view this on mobile. But a) I'm not sure what's causing the inconsistency and b) I personally think they would look 🤩 stacked in a single column on a small screen size.

Reply:
I use arbitrary value since this work does not really require UX attention:
Min_Screen_1&2: 1 cols
Min_Screen_3: 2 cols
Min_Screen_4+: 3 cols